usage-viewer/README.md

```diff
@@ -0,0 +1,71 @@
+# Retool Usage Viewer - Identify top apps by pageviews and queries
+
+## Requirements
+This app needs read access to [Retool's storage database](https://docs.retool.com/docs/configuring-retools-storage-database). Therefore, it is only suitable for use with self-hosted instances of Retool.
+
+## Why you need this
+As an administrator of a self-hosted Retool instance, you want to understand the most frequently used apps in your instance. The Retool audit log contains the information you need, but it's difficult to access and make sense of.
+
+The Retool Usage Viewer makes your job easier by displaying app usage in a way that's easy to consume.
+
+To track app usage beyond pageviews and queries, consider defining [analytics events](https://docs.retool.com/docs/analytics).
```

Comment:
This is a nice way to work in that detail. 
Can we add a phrase, "To track app usage beyond pageviews and queries, or over long time ranges, [...]"